app/src/main/kotlin/com/simplemobiletools/flashlight/dialogs/SleepTimerCustomAlertDialog.kt

```diff
@@ -0,0 +1,140 @@
+package com.simplemobiletools.flashlight.dialogs
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.KeyboardType
+import com.simplemobiletools.commons.R
+import com.simplemobiletools.commons.compose.alert_dialog.AlertDialogState
+import com.simplemobiletools.commons.compose.alert_dialog.rememberAlertDialogState
+import com.simplemobiletools.commons.compose.extensions.MyDevices
+import com.simplemobiletools.commons.compose.theme.AppThemeSurface
+import com.simplemobiletools.commons.dialogs.dialogBorder
+import com.simplemobiletools.commons.dialogs.dialogContainerColor
+import com.simplemobiletools.commons.dialogs.dialogElevation
+import com.simplemobiletools.commons.dialogs.dialogShape
+
+private val items = listOf(
+    R.string.minutes_raw,
+    R.string.seconds_raw,
+)
+
+@Composable
+fun SleepTimerCustomAlertDialog(
+    alertDialogState: AlertDialogState,
+    modifier: Modifier = Modifier,
+    onConfirmClick: (seconds: Int) -> Unit,
+    onCancelClick: () -> Unit = {}
```

Comment:
Better have it as a null, this way we're allocating unnecessary amount of overhead for the compiler

Reply:
`onCancelClick: (() -> Unit)?= null